openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx

```diff
@@ -15,10 +34,53 @@ const WaitingStepList = ({ studentList, refreshData }) => {
     <Button onClick={() => refreshData()}>{gettext('Refresh')}</Button>
   );
 
+  const FindLearnerAction = (props) => {
+    const { tableInstance: { selectedFlatRows = [] } } = props || { tableInstance: {} };
```

Comment:
I'm not sure what's selectedFlatRows or tableInstance. Can we use a better names?

Reply:
I can't change the name of those, are names that come from [DataTable paragon](https://paragon-openedx.netlify.app/components/datatable/) component